third_party/2and3/cryptography/x509/__init__.pyi

```diff
@@ -286,6 +286,14 @@ class Extensions(object):
     def get_extension_for_oid(self, oid: ObjectIdentifier) -> Extension: ...
     def get_extension_for_class(self, extclass: Type[_T]) -> Extension[_T]: ...
 
+class DuplicateExtension(Exception):
+    oid: ObjectIdentifier
+    def __init__(self, msg: str, oid: ObjectIdentifier) -> None: ...
+
+class ExtensionNotFound(Exception):
+    oid: ObjectIdentifier
+    def __init__(self, msg: str, oid: ObjectIdentifier) -> None: ...
+
```

Comment:
Not a full review, but in the implementation, these also live in `extensions.py` and are only re-exported from `__init__.py`.

Reply:
That's true, but implementation is very weird in this aspects, in contains most things in two places.

So I've checked the documentation and placed things where documentation places them.